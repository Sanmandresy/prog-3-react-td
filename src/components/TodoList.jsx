import React, { useState } from "react";
import Dones from "./Dones";
import Todos from "./Todos";

const TodoList = () => {
  const [todos, setTodos] = useState([]);
  return (
    <>
      <main className="main__screen">
        <Todos setTodos={setTodos} bridge={todos} />
        <Dones doneList={todos} />
      </main>
    </>
  );
};

export default TodoList;
