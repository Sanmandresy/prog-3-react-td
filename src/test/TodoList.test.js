import renderer from "react-test-renderer";
import TodoList from "../components/TodoList";

describe(TodoList, () => {
  it("should render the todo list component", () => {
    const component = renderer.create(<TodoList />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
