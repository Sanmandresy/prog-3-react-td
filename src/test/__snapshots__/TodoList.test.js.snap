// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoList should render the todo list component 1`] = `
<main
  className="main__screen"
>
  <section>
    <h1>
      TO DO
    </h1>
    <input
      className="todo__input"
      data-testid="input_todo"
      onChange={[Function]}
      onKeyDown={[Function]}
      placeholder="Add your todo..."
      type="text"
      value=""
    />
    <ul />
  </section>
  <section>
    <h2>
      DONE
    </h2>
    <ul />
  </section>
</main>
`;
